packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -262,9 +262,9 @@ class FlexibleSpaceBarSettings extends InheritedWidget {
   /// [FlexibleSpaceBar] widget in its tree for the settings to take affect.
   const FlexibleSpaceBarSettings({
     Key key,
-    this.toolbarOpacity,
-    this.minExtent,
-    this.maxExtent,
+    @required this.toolbarOpacity,
```

Comment:
The constructor's doc should say: The required [toolbarOpacity], [minExtent], and [maxExtent] parameters must not be null.

Reply:
Added this, not sure if it's in the correct place though

Reply:
That's perfect, thanks